HUD/GetFilenameForAudioConversation.md

```diff
@@ -0,0 +1,76 @@
+---
+ns: HUD
+aliases: ["0x7B5280EBA9840C72", _GET_LABEL_TEXT"]
+---
+## GET_FILENAME_FOR_AUDIO_CONVERSATION
+
+```c
+// 0x7B5280EBA9840C72 0x95C4B5AD
+char* GET_FILENAME_FOR_AUDIO_CONVERSATION(char* labelName);
```

Comment:
The name seems correct I generated the hash from the given native name and it matches `0x95C4B5AD`.